### PR TITLE
Update ci report_template.

### PR DESCRIPTION
**Pull request by huangmingxia**
@LiangquanLi930 @heliubj18 Could you help review it when you free? Thanks..

### CODE REVIEW

**Comment by huangmingxia** [comment]
@liangxia Could you help review it when you free? Thanks.

File: ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13-periodics.yaml [hunk 1]
```diff
@@ -37044,7 +37141,11 @@ periodics:
       job_states_to_report:
       - failure
       - error
-      report_template: Job {{.Spec.Job}} failed. <{{.Status.URL}}|View logs>
+      - success
```
Comment (liangxia) [inline ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13-periodics.yaml:37144]: Noticed that some jobs report `success` state, while not for the other jobs. Is it expected ?

Comment (huangmingxia) [reply ci-operator/jobs/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13-periodics.yaml:37144]: @liangxia Yes, it is our expect status.

**Pull request merged**